### PR TITLE
feat: :sparkles: .releaserc file added

### PR DESCRIPTION
**Pull request by bhavinbabariya03**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@distinction-dev/backend-package-name",
+  "name": "@distinction-dev/jasmine-console-reporter",
```
Comment (Jay-Motka-216) [inline package.json]: please remove "@distinction-dev"

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
   "version": "0.0.0-development",
   "description": "Template repository for us to use to create TS based backend packages",
```
Comment (Jay-Motka-216) [inline package.json]: Add apt description here for the repo

File: package.json [hunk 3]
```diff
@@ -57,6 +57,7 @@
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
     "chai": "^4.3.6",
+    "chalk": "4.1.2",
```
Comment (Jay-Motka-216) [inline package.json]: add this as a dependency and not a dev dependency

**Review by Jay-Motka-216 — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -84,5 +87,8 @@
     "branches": [
       "main"
     ]
+  },
+  "publishConfig": {
```
Comment (Jay-Motka-216) [inline package.json:92]: add the registry field in the publishConfig as well

File: package.json [hunk 5]
```diff
@@ -33,7 +33,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/distinction-dev/package-name.git"
+    "url": "git+https://github.com/distinction-dev/jasmine-console-reporter.git"
   },
   "license": "MIT",
   "author": {
```
Comment (Jay-Motka-216) [inline package.json:39]: change the author name to your name and details

File: src/index.ts [hunk 6]
```diff
@@ -1 +1,76 @@
...
+    );
+    console.log(
+      `Failure Spec Count : ${this.specCount - this.specSuccessCount} of ${
+        this.specCount
+      }`
+    );
+  }
+}
+
+export default CustomReporter;
```
Comment (Jay-Motka-216) [inline src/index.ts:76]: export the types here as well

File: src/types.ts
Comment (Jay-Motka-216) [inline src/types.ts:1]: add the types to src/types/index.ts file

**Review by Jay-Motka-216 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 6 above)
Comment (bhavinbabariya03) [reply src/index.ts:76]: I think there are no need to export type becuase we can give type as class name.

import MyCustomReporter from "./custom-reporter";
const customReporter: MyCustomReporter = new MyCustomReporter();

are you taking about any other types?

File: README.md [hunk 7]
```diff
@@ -1,26 +1,34 @@
...
+  showColors: true,
+});
+
+jasmine.execute();
 ```
 
 ## Docs
 
-Automatically generated docs are available [here](https://distinction-dev.github.io/package-name)
+Automatically generated docs are available [here](https://distinction-dev.github.io/jasmine-console-reporter)
```
Comment (Jay-Motka-216) [inline README.md]: remove this line as we do not have additional docs to show

**Review by Jay-Motka-216 — CHANGES_REQUESTED** [review]

**Review by Jay-Motka-216 — APPROVED** [review]

**Pull request merged**